### ISSUE TITLE
Writer not found for list inside dictionary with list<int> as value

### ISSUE THREAD

**Issue by TackJordy** [FirstGearGames/FishNet#232]
Unity version:
Fish-Networking version: 3.1.4

```

  [SyncObject]
    private readonly SyncDictionary<NetworkObject, List<int>> _vehicleOwners = new SyncDictionary<NetworkObject, List<int>>();
```
When using the following code this error appears: Write method not found for List`1. Use a supported type or create a custom serializer.

**Comment by FirstGearGames** [comment]
This may be fixed in 3.3.0. Let me know.

**Comment by FirstGearGames** [comment]
This is expected to be resolved in 3.3.1. POC working.

**Comment by FirstGearGames** [comment]
Tested many scenarios and working. If you're on the $10+ tier you can grab the latest now from #closed-testing on Discord.
